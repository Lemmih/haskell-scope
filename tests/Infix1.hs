module Infix1 where

a + b = b + a
