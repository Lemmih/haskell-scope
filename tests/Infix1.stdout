Scope errors:

Definitions:
  Definition 1, from Infix1.decl0:
    [4ma + b = b + a[0m
  Definition 2, from Infix1.decl0.match0:
    [4ma[0m + b = b + a
  Definition 3, from Infix1.decl0.match0:
    a + [4mb[0m = b + a

Use sites:
  Definition used: 1
    a [4m+[0m b = b + a
  Definition used: 3
    a + b = [4mb[0m + a
  Definition used: 1
    a + b = b [4m+[0m a
  Definition used: 2
    a + b = b + [4ma[0m
